[2018-07-01 18:36:37,005] [11] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 18:38:01,901] [10] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 18:41:27,095] [6] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 18:42:17,686] [6] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 21:02:59,398] [7] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 22:31:30,095] [7] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 22:40:42,780] [11] ERROR ErrorLog [(null)]  - System.InvalidOperationException: 支持“SqlServerDbContext”上下文的模型已在数据库创建后发生更改。请考虑使用 Code First 迁移更新数据库(http://go.microsoft.com/fwlink/?LinkId=238269)。
   在 System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClasse`1.<CreateInitializationAction>b__d()
   在 System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   在 System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   在 System.Data.Entity.Internal.LazyInternalContext.<InitializeDatabase>b__4(InternalContext c)
   在 System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   在 System.Data.Entity.Internal.LazyInternalContext.InitializeDatabase()
   在 System.Data.Entity.Internal.InternalContext.Initialize()
   在 System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   在 System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   在 System.Data.Entity.Internal.Linq.InternalSet`1.GetEnumerator()
   在 System.Data.Entity.Infrastructure.DbQuery`1.System.Collections.Generic.IEnumerable<TResult>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 36
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 23:22:24,253] [6] ERROR ErrorLog [(null)]  - System.InvalidOperationException: Mapper already initialized. You must call Initialize once per application domain/process.
   在 AutoMapper.Mapper.set_Configuration(IConfigurationProvider value)
   在 AutoMapper.Mapper.Initialize(Action`1 config)
   在 MyFX.Core.AutoMapperExtension.MapToList[TSource,TDestination](IEnumerable`1 source) 位置 d:\gitroot\MyFX\MyFX.Core\AutoMapperExtension.cs:行号 26
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 39
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 23:22:37,100] [6] ERROR ErrorLog [(null)]  - System.InvalidOperationException: Mapper already initialized. You must call Initialize once per application domain/process.
   在 AutoMapper.Mapper.set_Configuration(IConfigurationProvider value)
   在 AutoMapper.Mapper.Initialize(Action`1 config)
   在 MyFX.Core.AutoMapperExtension.MapToList[TSource,TDestination](IEnumerable`1 source) 位置 d:\gitroot\MyFX\MyFX.Core\AutoMapperExtension.cs:行号 26
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 39
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
[2018-07-01 23:23:16,564] [7] ERROR ErrorLog [(null)]  - System.InvalidOperationException: Mapper already initialized. You must call Initialize once per application domain/process.
   在 AutoMapper.Mapper.set_Configuration(IConfigurationProvider value)
   在 AutoMapper.Mapper.Initialize(Action`1 config)
   在 MyFX.Core.AutoMapperExtension.MapToList[TSource,TDestination](IEnumerable`1 source) 位置 d:\gitroot\MyFX\MyFX.Core\AutoMapperExtension.cs:行号 26
   在 DC.Service.Core.FindTablesCore.Execute() 位置 e:\mytest\DataCenter\0-Core\DC.Service\Core\FindTablesCore.cs:行号 39
   在 MyFX.Core.Actions.ServiceOptionBase`2.DoExecute(Boolean throwException) 位置 d:\gitroot\MyFX\MyFX.Core\Actions\ServiceOptionBase.cs:行号 51
